wordpress-mobile/gutenberg-mobile
# Enables native image fullscreen preview on iOS

### Fixes #1286 

`WordPress-iOS` PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/12979
`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/18618

### Description
This change enables a call to native fullscreen image previews on iOS from the mobile image block. It also fixes some minor UX issues around not highlighting the image while caption is selected and not launching fullscreen image on tap when caption is selected (See https://github.com/wordpress-mobile/gutenberg-mobile/issues/1286#issuecomment-554525746 ).

### Testing
1. Navigate to edit a post in the Block Editor.
2. Add an image block and upload an image from any source.
3. Click on the image once to show a blue border highlighted state.
4. Click the image while the image block is selected and verify the preview image screen is shown.
5. Update release notes:

![ios-fullscreen-video](https://user-images.githubusercontent.com/1103838/69202284-0926ed00-0af6-11ea-8dd3-781d2623e6ad.gif)
